src/utils.coffee

```diff
@@ -44,13 +44,18 @@ class Utils
     result = [result] unless result instanceof Array
     _.compact(result)
 
-  @wrapObjects: (array) ->
+  @wrapObjects: (array, options = {}) ->
     output = []
     _(array).each (elem) =>
       if elem.constructor == Object
         for key, value of elem
           o = {}
-          o[key] = @wrapObjects(if value instanceof Array then value else [value])
+
+          if options.ignoreCollections && elem[key] instanceof Backbone.Collection
```

Comment:
I see. Yeah, I guess this internal method doesn't need documentation. I was thinking it was being passed from the public API.

A couple of questions: Should we always be ignore collections? 

I mean, this method is used to normalize the options passed around.

e.g.

```
include: { 'foo': 'bar' }
```

gets converted to

```
include: [{ foo: ['bar'] }]
```

because an association can have many sub-associations. 

How is this collection stuff any different? 

Is the following not valid?

```
include: { 'foo': [someCollection] }
```